### ISSUE TITLE
WCAG audit and improvements

### ISSUE THREAD

**Issue by krzysztofrewak** [blumilksoftware/keating#124]
I would like to check if everything here is good accordingly to current WCAG. This gonna be website for teachers, so we need to apply all accesibility features.

**Comment by Blusia** [comment]
**This comment will be edited when I find things related to accessibility.**

- [ ] I think that in the place where the link of the page is located, e.g. Blumilk or repo Keating there should be a "label" telling us to open a new tab, e.g. "will open in a new window". Each place that, when clicked by the user, opens a new tab must be appropriately marked.
![obraz](https://github.com/user-attachments/assets/c1cf9e82-9220-4d2c-bbc3-8bbf5adfa7e6)

- [ ] Each available image must have an appropriate "alt" description. Decorative images should have an empty `alt=""` attribute so that the reader can skip the image without reading the description. Missing alt on main login page - university logo.
`<img src="https://dev.keating.blumilk.pl/cwup-full.png" class="mx-auto w-[360px]">`
![obraz](https://github.com/user-attachments/assets/6fcd25b0-d929-490f-b737-676e1fb7290d)


- [ ] Customize the link so that users can skip repetitive elements like the navigation panel, etc. When I click on the tab on the grades page, I select a semester, and when I want to select the next semester, the tab takes me through the entire navigation panel again.  _(btw. I still need to verify this)_

- [ ] Zooming and scaling shouldn't be disabled. Maximum-scale on `<meta>` tag disables zooming on mobile devices. Currently browsers like chrome on android allow you to force the page to be zoomed in, but these are additional settings that need to be enabled by user.

After logging in:
Dashboard:

- [ ] links and buttons must have accesible text that is visible to screen readers.

- [ ] There is also a problem with color contrast for parts such as "Uczelnia", "Słowniki", "Ogólne", "Dashboard" (Referring to the word "Dashboard", maybe it would be a good idea to use a Polish equivalent here? Like "Panel"?)
